Frontend/ui-customer/src/Resources/Localizable/BN.purs

```diff
@@ -658,4 +658,54 @@ getBN stringKey =
         FARE_UPDATED_WITH_CHARGES_SHORTER_DIST -> "আপনার ভাড়া আপডেট করা হয়েছে কারণ আপনার রাইড {} কিমি কম ছিল এবং অপেক্ষার সময় চার্জ নেওয়া হয়েছে"
-        FARE_UPDATED_WITH_CHARGES_LONGER_DIST -> "আপনার ভাড়া আপডেট করা হয়েছে কারণ আপনার যাত্রা {} কিমি দীর্ঘ ছিল এবং অপেক্ষার সময় চার্জ নেওয়া হয়েছে"
\ No newline at end of file
+        FARE_UPDATED_WITH_CHARGES_LONGER_DIST -> "আপনার ভাড়া আপডেট করা হয়েছে কারণ আপনার যাত্রা {} কিমি দীর্ঘ ছিল এবং অপেক্ষার সময় চার্জ নেওয়া হয়েছে"
+        YOUR_REPORTS -> "আপনার রিপোর্ট"
+        VIEW -> "দেখুন"
+        ISSUE_NO -> "ইস্যু নং"
+        ADD_VOICE_NOTE -> "ভয়েস নোট যোগ করুন"
+        VOICE_NOTE -> "ভয়েস নোট"
+        SUBMIT_ISSUE_DETAILS -> "ইস্যু বিবরণ জমা দিন"
+        IMAGE_PREVIEW -> "ছবির পূর্বরূপ"
+        ADDED_IMAGES -> "ছবি যোগ করা হয়েছে"
+        NO_IMAGES_ADDED -> "কোন ছবি যোগ করা হয়নি"
+        I_DONT_KNOW_WHICH_RIDE -> "আমি জানি না কোন রাইড"
+        ADDED_VOICE_NOTE -> "ভয়েস নোট যোগ করা হয়েছে"
+        NO_VOICE_NOTE_ADDED -> "কোন ভয়েস নোট যোগ করা হয়নি"
+        CALL_DRIVER_TITLE -> "ড্রাইভারকে ডাকবেন?"
+        CALL_SUPPORT_TITLE -> "কল সমর্থন?"
+        CALL_DRIVER_DESCRIPTION -> "আপনি ড্রাইভারকে একটি কল করতে চলেছেন৷ আপনি কি এগিয়ে যেতে চান?"
+        ADD_IMAGE -> "ছবি যোগ কর"
+        ADD_ANOTHER -> "আরেকটি যোগ করুন"
+        IMAGES -> "ছবি"
+        ISSUE_SUBMITTED_TEXT -> "অপেক্ষা কর! আমরা আপনার সমস্যা সমাধানে কাজ করছি"
+        ISSUE_RESOLVED_TEXT -> "ব্যাপারতি পুনঃসংশোধিত!"
+        CHOOSE_AN_OPTION -> "চালিয়ে যাওয়ার জন্য একটি বিকল্প বেছে নিন"
+        ISSUE_MARKED_AS_RESOLVED -> "সমস্যা সমাধান হিসাবে চিহ্নিত!"
+        STILL_HAVING_ISSUE -> "এখনও একটি সমস্যা আছে?"
+        IMAGE -> "ছবি"
+        RECORD_VOICE_NOTE -> "ভয়েস নোট রেকর্ড করুন"
+        CANCEL_BUTTON -> "বাতিল করুন"
+        MAX_IMAGES -> "সর্বোচ্চ ৩টি ছবি আপলোড করা যাবে"
+        SELECT_A_RIDE_TO_REPORT -> "সমস্যা রিপোর্ট করতে রাইড নির্বাচন করুন"
+        RIDE_RELATED_ISSUE_PAGE_NAME -> "রাইড সম্পর্কিত সমস্যা"
+        APP_RELATED_ISSUE_PAGE_NAME -> "অ্যাপ সম্পর্কিত সমস্যা"
+        DRIVER_RELATED_ISSUE_PAGE_NAME -> "ড্রাইভার সম্পর্কিত সমস্যা"
+        LOST_AND_FOUND_ISSUE_PAGE_NAME -> "হারানো আইটেম রিপোর্ট করুন"
+        YEARS_AGO -> "অনেক বছর আগে"
+        MONTHS_AGO -> "মাস কতক পূর্বে"
+        DAYS_AGO -> "দিন আগে"
+        HOURS_AGO -> "ঘন্টা আগে"
+        MIN_AGO -> "মিনিট আগে"
+        SEC_AGO -> "এই মাত্র"
+        REPORTED -> "রিপোর্ট করেছেন"
+        RESOLVED -> "সমাধান করা হয়েছে"
+        RESOLVED_ISSUES -> "সমাধান করা সমস্যা"
+        REPORTED_ISSUES -> "রিপোর্ট করা সমস্যা"
+        OTHER_ISSUES -> "অন্যান্য সমস্যা"
+        SOS_ISSUE_PAGE_NAME -> "SOS সমস্যা"
+        FARE_DISCREPANCIES_ISSUE_PAGE_NAME -> "ভাড়া সংক্রান্ত সমস্যা"
+        PAYMENT_RELATED_ISSUE_PAGE_NAME -> "পেমেন্ট সম্পর্কিত সমস্যা"
+        ACCOUNT_RELATED_ISSUE_PAGE_NAME -> "অ্যাকাউন্ট সম্পর্কিত সমস্যা"
+        REPORT_ISSUE_CHAT_PLACEHOLDER _ -> ""
```

Comment:
If it is coming from config then also better to have default case strings,
In case of config fallback.
Please assign some value to these strings